### PR TITLE
Create layout for episode page

### PR DESCRIPTION
**Pull request by BPScott**
Basic columns are present, along with TODO blocks for every module

---

This supersedes #435

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Ds2013/Css/Core/Gel/Objects/_grid.scss [hunk 1]
```diff
@@ -210,3 +210,30 @@ $massive-pushdown: 50em;
...
     }
 }
+
+// Bound grids
+
+// When a single column looks too wide but two side-by-side columns looks
+// too skinny use a grid--bounded to have a single column with some spacing
+// around it (best used with grid-wrapper--center).
+.grid--bounded {
+    max-width: 624px;
```
Comment (votemike) [inline src/Ds2013/Css/Core/Gel/Objects/_grid.scss]: Do we have some sort of constant/variable for this number?

File: templates/find_by_pid/episode.html.twig [hunk 2]
```diff
@@ -1,24 +1,58 @@
...
+
+{# TODO Episode Panels #}
+
+<div class="prog-layout programmes-page">
+    <div class="grid-wrapper grid-wrapper--center">
+        {# TODO Related Topics Module #}
+
+        <div class="grid grid--bounded 13/24@bpw2 13/24@bpe">
+            <div class="br-box-page prog-box">
+                {# TODO Live Text Stream #}
```
Comment (tatitati) [inline templates/find_by_pid/episode.html.twig]: Now I catch your idea about TODO, very nice 👍

File: src/Ds2013/Css/Core/Gel/Objects/_grid.scss (hunk 1 above)
Comment (BPScott) [reply src/Ds2013/Css/Core/Gel/Objects/_grid.scss]: No, it isn't used elsewhere / there are no other numbers that hang off it IIRC

**Pull request merged**